draft-ietf-quic-http.md

```diff
@@ -825,6 +821,70 @@ MUST be treated as a connection error of type HTTP_MALFORMED_FRAME.
 A server MUST treat a MAX_PUSH_ID frame payload that does not contain a single
 variable-length integer as a connection error of type HTTP_MALFORMED_FRAME.
 
+### INITIAL_PRIORITY {#frame-initial-priority}
+
+The INITIAL_PRIORITY (type=0x0F) frame specifies the client-advised priority of
+a new stream.  INITIAL_PRIORITY frames MAY be sent as the first frame of a
+request stream, and MUST NOT be sent thereafter or on other streams.  An
+INITIAL_PRIORITY frame sent on a non-request stream MUST be treated as a
+connection error of type HTTP_WRONG_STREAM.  An INITIAL_PRIORITY frame received
+after other frames or received by a client MUST be treated as a stream error of
+type HTTP_UNEXPECTED_FRAME.
+
+~~~~~~~~~~  drawing
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|DT |   Empty   |           Element Dependency ID (i)         ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|   Weight (8)  |
++-+-+-+-+-+-+-+-+
+~~~~~~~~~~
+{: #fig-initial-priority title="INITIAL_PRIORITY frame payload"}
+
+The INITIAL_PRIORITY frame payload has the following fields:
+
+  Dependency Type:
+  : A two-bit field indicating the type of element being depended on.
+
+  Empty:
+  : A six-bit field which MUST be zero when sent and MUST be ignored
+    on receipt.
+
+  Element Dependency ID:
+  : A variable-length integer that identifies the element on which a dependency
+    is being expressed. Depending on the value of Dependency Type, this contains
+    the Stream ID of a request stream, the Push ID of a promised resource, the
+    Placeholder ID of a placeholder, or is ignored.  For details of
+    dependencies, see {{priority}} and {{!RFC7540}}, Section 5.3.
```

Comment:
No I'm confused by this EDID. The frame is only applicable to request streams, so I think the EDID would always equal stream ID. It seems redundant to send this information twice.